src/geo/ui/attribution/attribution-view.js

```diff
@@ -23,14 +23,12 @@ var Attribution = View.extend({
       visible: false
     });
     this.map = this.options.map;
-    this.model.bind('change:visible', function (mdl, isVisible) {
-      this[ isVisible ? '_showAttributions' : '_hideAttributions' ]();
-    }, this);
-    _.bindAll(this, '_onKeyDown', '_toggleAttributions');
+
```

Comment:
Out of curiosity... why are we moving from ☝️ to 👇 ? I kind of like `_.bindAll` better...